### DISCUSSION TITLE
Templatable attributes

### DISCUSSION THREAD

**Discussion by IgorSimic** [kinghat/tabbed-card#136] [Feature Requests]
Hi,
Please, is it possible to make attributes templatable, at least icon and label?
I would like to use icon and/or label to indicate state of something.
Example:
In one tab I have thermostat for my heating, in another for air conditioner.
With template, I would like change icon to reflect states of them (on/off).
Thanks in advance.

**Comment by kinghat** [comment]
already requested here: https://github.com/kinghat/tabbed-card/discussions/94
